package.json

```diff
@@ -146,17 +146,17 @@
     "@testing-library/user-event": "^14.4.3",
     "@types/jest": "^29.5.4",
     "@types/js-cookie": "^3.0.3",
-    "@types/node": "^17.0.45",
-    "@types/react": "^18.0.25",
+    "@types/node": "^20.5.9",
```

Comment:
this should match the Node version we are using, so needs to be the latest 18.x version

Reply:
I've approved ahead of the Node version getting lined up. If the change doesn't automatically dismiss this thread you should be able to click "Resolve conversation"

Reply:
Changed to 18.11.9